### PR TITLE
added SharedPreferences save User Profile

### PR DESCRIPTION
**Pull request by mkenlo**
I added shared preferences to save user profile, then I updated the author field when I create a new translation object.

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -14,7 +14,7 @@ class MyApp extends StatelessWidget {
       theme: ThemeData(
         fontFamily: "Raleway",
         primarySwatch: Colors.teal,
-        brightness: Brightness.light,
+        //brightness: Brightness.light,
```
Comment (evindj) [inline lib/main.dart:17]: Nit: instead of adding  a comment we should just remove if we see fit.

File: lib/services/fb_login.dart [hunk 2]
```diff
@@ -2,12 +2,26 @@ import "dart:async";
...
+  String queryFields = "name,first_name,last_name,email,location,picture";
 
   final graphResponse = await http.get(
       "https://graph.facebook.com/v4.0/me?fields=$queryFields&access_token=$accessToken");
-  return json.decode(graphResponse.body);
+  final jsonProfile = json.decode(graphResponse.body);
+  return User.fromJson(jsonProfile);
+}
+
+Future saveUserProfile(User user) async {
```
Comment (evindj) [inline lib/services/fb_login.dart:19]: Why don't you specify the return type of the future? I was expecting something like Future<int>?

**Review by evindj — APPROVED** [review]
Please look at my comments before merging. Fix them if they need fixing.

File: lib/main.dart (hunk 1 above)
Comment (mkenlo) [reply lib/main.dart:17]: Oh! Just forgot to remove. Will do in next PR

File: lib/services/fb_login.dart (hunk 2 above)
Comment (mkenlo) [reply lib/services/fb_login.dart:19]: I will add a specific type for Future in this next PR

**Pull request merged**